app/src/main/java/com/twilio/voice/quickstart/VoiceActivity.java

```diff
@@ -252,7 +261,6 @@ private void handleIncomingCallIntent(Intent intent) {
                             answerCallClickListener(),
                             cancelCallClickListener());
                     alertDialog.show();
-                    notificationManager.cancel(intent.getIntExtra(INCOMING_CALL_NOTIFICATION_ID, 0));
```

Comment:
Should this still be called? Perhaps right after `accept()` or `reject()` are clicked? Otherwise doesn't the notification stay in the notification drawer?

Reply:
When the call is accepted or rejected , the app gets `cancel` notification and it gets cleared at that point.